modules/core/04-channel/keeper/packet_test.go

```diff
@@ -773,7 +773,7 @@ func (suite *KeeperTestSuite) TestAcknowledgePacket() {
 
 			channelCap = suite.chainA.GetChannelCapability(path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 		}, true},
-		{"success on channel in tryupgrade and flush status in flushing", func() {
+		{"success on channel in flushing state", func() {
```

Comment:
At the moment this is functioning with "tryupgrade" rather than the flushing state, but I made the change to the test string because this will still pass successfully when https://github.com/cosmos/ibc-go/issues/4243 is completed